README.md

```diff
@@ -10,17 +10,28 @@ Self-documenting Node scripts through literate programming. Based on [lit](https
 
 ```bash
 # install
-$ npm install --global lit-node
+$ npm install lit-node
+
+# load module from command line to enable
+# execution of literate Markdown with Node
+$ node --require lit-node program.md
```

Comment:
are we gonna do that thing where index.js exports the API, and you have to `-r lit-node/register` to register the .md extension?

```suggestion
$ node --require lit-node/register program.md
```